### PR TITLE
Describe about menu customization

### PR DESCRIPTION
**Pull request by toshi-k**
Cluster admin can customize the menu bar by ConfigMap from KF1.3 (some parts of this feature was already available in KF1.2).
Since it is not written in the current documentation, this PR add a document about it.

- Centraldashboard: update menu items in manifests #5706<br>https://github.com/kubeflow/kubeflow/pull/5706

In addition, recent update improved the capability to support namespaced menu items. I wrote it about it too.

- Make it possible to add Namespaced menu items #5871<br>https://github.com/kubeflow/kubeflow/pull/5871

(Since I'm not a native English speaker, I appreciate if some reviewers correct unnatural sentences.)

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RFMVasconcelos

**Comment by thesuperzapper** [comment]
@toshi-k I believe this sort of information should go under the `Components / Central Dashboard` section, rather than `Further Setup and Troubleshooting`:
https://www.kubeflow.org/docs/components/central-dash/

**Comment by toshi-k** [comment]
@thesuperzapper I've never seen that page before, but I totally agree with you. I moved the new document to the section for the central dashboard. 

@RFMVasconcelos Could you review or LGTM my latest PR ?

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you @toshi-k for this. Added a few comments :)

File: content/en/docs/components/central-dash/customizing-menu.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+The universal way to handle this problem is deploying the app for each namespace.
+The cluster admin deploy the app for each namespace and URLs would be like below.
+```
+http(s)://gateway/_/myapp/profile1/
+http(s)://gateway/_/myapp/profile2/
+```
+
+In this case, you can configure the central dashboard as below.
+`{ns}` is replace by the namespace when the user open the dashboard.
```
[the remark this answers is not in the document]
Comment (toshi-k) [reply content/en/docs/components/central-dash/customizing-menu.md]: Fixed

File: content/en/docs/components/central-dash/customizing-menu.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          "icon": "social:mood"
+        }
+      ],
+```
+
+The users can see a new item (in this case, it is MyApp as well) on the menubar.
+They can either jump to `http(s)://gateway/_/myapp/profile1/` or `http(s)://gateway/_/myapp/profile2/` based on the namespace selection.
+The actual inside content of iframe is swiched by the namespace. 
+
+If sidecar injection is enabled, the authrization to the app is done by istio.
```
[the remark this answers is not in the document]
Comment (toshi-k) [reply content/en/docs/components/central-dash/customizing-menu.md]: Fixed

File: content/en/docs/components/central-dash/customizing-menu.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+e.g) The uses who don't belong to profile2 can't access to `http(s)://gateway/_/myapp/profile2/`.
```
[the remark this answers is not in the document]
Comment (toshi-k) [reply content/en/docs/components/central-dash/customizing-menu.md]: Fixed

File: content/en/docs/components/central-dash/customizing-menu.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+In a below example, "My App" is added on the side menubar.
+
+<img src="/docs/images/customize-menu-add-app.png" 
+  alt="Display third party app on a kubeflow dashboard"
+  class="mt-3 mb-3 border border-info rounded">
+
+## Add shared items
+The way to add items shared by all users is described in this section.
+
+First, the cluster admin deploys an app as a service of the Kubernetes.
```
[the remark this answers is not in the document]
Comment (toshi-k) [reply content/en/docs/components/central-dash/customizing-menu.md]: Fixed

**Comment by toshi-k** [comment]
@RFMVasconcelos Thank you for your review. I updated my PR as you proposed. Please review my latest PR again.

**Comment by RFMVasconcelos** [comment]
Awesome @toshi-k ! 

Thank you for contributing this to the community :) 

/lgtm
/approve

**Pull request merged**